Realm/RLMSyncConfiguration.mm

```diff
@@ -270,32 +285,43 @@ - (BOOL)enableFlexibleSync {
     return make_sync_error(errorClass, @(error.message.c_str()), error.error_code.value(), custom);
 }
 
-static void setDefaults(SyncConfig& config, RLMUser *user) {
-    config.client_resync_mode = ClientResyncMode::Manual;
-    config.stop_policy = SyncSessionStopPolicy::AfterChangesUploaded;
-
+- (void)setErrorHandler:(RLMUser *)user {
```

Comment:
`setErrorHandler:` is a bad name for this function, as it's not a setter for the `errorHandler` property.